### PR TITLE
Convert exceptions to Django exceptions

### PR DESCRIPTION
**Pull request by noisecapella**

### CODE REVIEW

**Comment by ShawnMilo** [comment]
This is unnecessary; the exact code was just removed in #176 when the decorator was added. Discussion in that PR led to removal because it leads to redundant code which could never be hit, and also reduce test coverage.

**Comment by noisecapella** [comment]
`get_repo` raises `PermissionDenied` so `get_resource` will also raise `PermissionDenied` since it calls `get_repo`

**Comment by ShawnMilo** [comment]
@giocalitri, I believe that the decorator prevents any of these from ever happening. Is that correct?

**Comment by giocalitri** [comment]
in that specific case yes  and in general in any case where the user does not have view permissions on the repo

**Comment by noisecapella** [comment]
Ok, I've removed the except clause

File: ui/views.py [hunk 1]
```diff
@@ -137,7 +137,10 @@ def upload(request, repo_slug):
     """
     Upload a OLX archive.
     """
-    repo = get_repo(repo_slug, request.user.id)
+    try:
+        repo = get_repo(repo_slug, request.user.id)
```
Comment (ShawnMilo) [inline ui/views.py]: Everything looks good except these additions -- the decorator means it can't happen.

**Pull request merged**